server/routerlicious/packages/lambdas/src/nexus/trace.ts

```diff
@@ -3,7 +3,6 @@
  * Licensed under the MIT License.
  */
 
-import { performance } from "@fluidframework/common-utils";
```

Comment:
Server code only runs in node, it can use the native `performance.now()` directly, no need to try to proxy through an isomorphic performance object (which incidentally was already removed in main for the client release group).